rct229/ruleset_functions/baseline_systems/test_is_baseline_system_13.py

```diff
@@ -0,0 +1,151 @@
+                                    "terminals": [
+                                        {
+                                            "id": "CAV Air Terminal 1",
+                                            "is_supply_ducted": True,
+                                            "type": "CONSTANT_AIR_VOLUME",
+                                            "served_by_heating_ventilating_air_conditioning_system": "System 13",
+                                        },
+                                    ],
+                                },
+                                {
+                                    "id": "Thermal Zone 2",
+                                    "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                    "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                    "terminals": [
+                                        {
+                                            "id": "CAV Air Terminal 2",
+                                            "is_supply_ducted": True,
+                                            "type": "CONSTANT_AIR_VOLUME",
+                                            "served_by_heating_ventilating_air_conditioning_system": "System 13s",
+                                        },
+                                    ],
+                                },
+                            ],
+                            "heating_ventilating_air_conditioning_systems": [
+                                {
+                                    "id": "System 13",
+                                    "cooling_system": {
+                                        "id": "CHW Coil 1",
+                                        "cooling_system_type": "FLUID_LOOP",
+                                        "chilled_water_loop": "Secondary CHW Loop 1",
+                                    },
+                                    "heating_system": {
+                                        "id": "Heating Coil 1",
+                                        "heating_system_type": "ELECTRIC_RESISTANCE",
+                                    },
+                                    "fan_system": {
+                                        "id": "CAV Fan System 1",
+                                        "fan_control": "CONSTANT",
+                                        "supply_fans": [{"id": "Supply Fan 1"}],
+                                        "return_fans": [{"id": "Return Fan 1"}],
+                                    },
+                                },
+                                {
+                                    "id": "System 13a",
+                                    "cooling_system": {
+                                        "id": "CHW Coil 1",
+                                        "cooling_system_type": "FLUID_LOOP",
+                                        "chilled_water_loop": "Chilled Water Loop 1",
```

Comment:
If this is purchased chilled water, suggest to change the loop ID to a more explicitly name -> **Purchased CHW Loop 1** for example.

Reply:
Great catch! I just addressed this.